slick-testkit/src/codegen/resources/dbs/sqlite.sql

```diff
@@ -1,2 +1,3 @@
 CREATE TABLE "SUPPLIERS" ("SUP_ID" INTEGER NOT NULL,"SUP_NAME" VARCHAR(254) NOT NULL,"STREET" VARCHAR(254) NOT NULL,"CITY" VARCHAR(254) NOT NULL,"STATE" VARCHAR(254) NOT NULL,"ZIP" VARCHAR(254) NOT NULL);
 CREATE TABLE "COFFEES" ("COF_NAME" VARCHAR(254) NOT NULL,"SUP_ID" INTEGER NOT NULL,"PRICE" DOUBLE NOT NULL,"SALES" INTEGER NOT NULL,"TOTAL" INTEGER NOT NULL,constraint "SUP_FK" foreign key("SUP_ID") references "SUPPLIERS"("SUP_ID") on update NO ACTION on delete NO ACTION);
+CREATE TABLE "TIMESTAMP"("TS" TIMESTAMP NOT NULL DEFAULT "2016-07-29 09:14:56");
```

Comment:
Does it make sense to add more tests on different time strings format allowed by SQLite?